packages/fhir-care-team/src/components/CreateEditCareTeam/Form.tsx

```diff
@@ -42,6 +42,11 @@ const CareTeamForm: React.FC<CareTeamFormProps> = (props: CareTeamFormProps) =>
   const { t } = useTranslation();
   const [form] = Form.useForm();
 
+  const effectInitalValuesDep = JSON.stringify(initialValues);
+  useEffect(() => {
+    form.resetFields();
```

Comment:
Is this not happening by default? Could it be a re-render issue on the parent component?

Reply:
let me recheck.

Reply:
fixed in [c27e113](https://github.com/opensrp/web/pull/1175/commits/c27e11302d108368d7d7dcefa4c1f8152e1f5517)